webapp/channels/src/components/post/post_options.tsx

```diff
@@ -25,11 +25,10 @@ type Props = {
     post: Post;
     teamId: string;
     isFlagged: boolean;
-    removePost: (post: Post) => void;
+    removePostCloseRHSDeleteDraft: (post: Post) => void;
```

Comment:
0/5 on changing it back, but I do like keeping the old name for actions like this. I don't think that the name of an action passed to the component needs to fully explain what it does as long as it's clear which button on the component it's triggered by.

Reply:
yes, that makes sense. I changed it back